code/game/objects/items/stacks/stack.dm

```diff
@@ -457,6 +458,10 @@
 
 	return TRUE
 
+/// Called after an item is constructed.
+/atom/proc/on_constructed(mob/builder)
+	return
```

Comment:
I still think this should be a signal and you should say why you don't think so

Reply:
because i was lazy

Reply:
well it'd be a super duper easy to implement signal so go for it